config/SLUS_005.61.splat.yaml

```diff
@@ -31,6 +31,7 @@ segments:
     type: code
     start: 0x00000800
     vram:  0x80010000
+    bss_size: 0x46B20
```

Comment:
I thought something like
```
      - [0xDEC0C, data]
      - {type: bss, vram: 0x8012F418}
```
was needed too but doesn't seem to make any difference

Reply:
Ah I think you're right. Without this, it might not auto-group symbols at the very end of the bss section.